src/test/scala/io/iohk/ethereum/vm/CallOpcodesSpec.scala

```diff
@@ -656,4 +670,42 @@ class CallOpcodesSpec extends WordSpec with Matchers {
     }
   }
 
+
+
+  "CallOpCodes" when {
+
+    Seq(CALL, CALLCODE, DELEGATECALL).foreach { opCode =>
+
+      s"$opCode processes returned data" should {
+
+        "handle memory expansion properly" in {
+
+          val inputData = ByteString(Array[Byte](1).padTo(32, 1.toByte))
+          val context: PC = fxt.context.copy(world = fxt.worldWithReturnSingleByteCode)
+
+          val outOffsets = Seq(0, inputData.size / 2, inputData.size * 2)
```

Comment:
I would recommend using a `Table` here, even if it's just a single column. A `Table` makes it easier to see which input caused a test failure.